src/iexamine.cpp

```diff
@@ -4141,10 +4173,10 @@ void iexamine::pay_gas( player &p, const tripoint &examp )
             return;
         }
 
-        int maximum_liters = std::min( money / pricePerUnit, tankGasUnits / 1000 );
+        int maximum_liters = std::min( money / pricePerUnit, tankUnits / 1000 );
 
-        std::string popupmsg = string_format(
-                                   _( "How many liters of gasoline to buy?  Max: %d L.  (0 to cancel)" ), maximum_liters );
+        std::string popupmsg = str_to_illiterate_str( string_format(
```

Comment:
Can someone check the addition of str_to_illiterate_str on the purchase message please? It feels like it was simply missing all along, but maybe there was a reason that specific string wasnt passed through str_to_illiterate_str?

Reply:
Hard to say - that was 6 years ago (https://github.com/CleverRaven/Cataclysm-DDA/pull/8137).